torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -595,10 +650,48 @@ def _annotate_output_for_int8_in_int8_out_pattern(
                     input_act = maxpool_node.args[0]
                     assert isinstance(input_act, Node)
                     assert isinstance(maxpool_node, Node)
-                    edge_or_node: Tuple[Node, Node] = (input_act, maxpool_node)
+                    edge_or_node = (input_act, maxpool_node)
                     getitem_node.meta[
                         "quantization_annotation"
                     ].output_qspec = SharedQuantizationSpec(edge_or_node)
+            elif node.target is torch.ops.aten.cat.default:
+                cat_node = node
+                cat_quantization_annotation = (
+                    cat_node.meta["quantization_annotation"]
```

Comment:
Is it better to build a constant variable for the key here?

Reply:
Not sure if my understanding is right or not. Did you mean define `cat_quantization_annotation 
 = cat_node.meta["quantization_annotation"]` and use `cat_quantization_annotation` later instead of `cat_node.meta["quantization_annotation"]`?

Reply:
Oh, I mean using some constant variable like `QUANT_ANNOTATION_KEY` instead of explicitly using `"quantization_annotation"` everywhere.

Reply:
Sure, make sense and changed. Please help to take a look again.